utils/utils.py

```diff
@@ -121,11 +121,19 @@ async def run_orion(
             "--config", config,
             "-o", "json"
         ]
+
+    es_metadata_index = os.environ.get("es_metadata_index")
+    if es_metadata_index == "" :
```

Comment:
nit, can you make it 
es_metadata_index = os.environ.get("ES_METADATA_INDEX")
same for benchmark index

Reply:
you want it in caps?

Reply:
yes
we can have a .env file and throw in all the vars there